module/VuFind/src/VuFind/ILS/Driver/KohaRest.php

```diff
@@ -2013,12 +2014,26 @@ protected function getItemStatusesForBiblio($id, $patron = null)
                 $duedate = null;
             }
 
+	    $extraStatusInformation = [];
+            if ($transit = $avail['unavailabilities']['Item::Transfer'] ?? null) {
+                if (null !== ($toLibrary = $transit['to_library'] ?? null)) {
+		    $status = 'transit_to';
```

Comment:
Since, you moved the translation to  'HoldingStatus.ini' you have to prefix the status with  'HoldingStatus::' here.